setup.py

```diff
@@ -90,7 +90,9 @@ def setup_environmental_variables():
 
     with open("./web/.env", "r+") as file:
         file_data = file.read()
-        file_data = re.sub(r"NEXT_PUBLIC_SUPABASE_URL=.*", f"NEXT_PUBLIC_SUPABASE_URL={supabase_anon_key}", file_data)
+        # required for supabase middleware fix
```

Comment:
```suggestion
        # required for supabase middleware fix
```
Oh great catch! Quick nit though, please don't add stray comments like this as this will be irrelevant after a couple of new commits.